docs/ARMInstallationManagedIdentity.md

```diff
@@ -51,7 +51,7 @@ The following parameters are provided by the ARM template:
 |**LookupWithEncounter**|Like the first setting but allows you to include an encounter identifier with the message to associate with the device/patient.  The encounter is looked up during processing and any observations created are linked to the encounter. The association here is assumed to be one encounter per device.
 
 ## Post Deployment
-After the ARM template is successfully deployed, add the Managed Identity ID output by the ARM deployment to the Allowed Object IDs on the Authentication page of your Azure API for FHIR. This is the identity in Azure Active Directory that the IoMT FHIR Connector for Azure uses to connect to Azure API for FHIR. Also, the Authority on the Authentication page of your Azure API for FHIR should NOT be changed from the Azure Active Directory in your subscription, or this connection will be broken.
+After the ARM template is successfully deployed, [configure Azure RBAC for your Azure API for FHIR](https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir/configure-azure-rbac) using the Managed Identity ID output by the ARM deployment in order to assign access for the IoMT FHIR Connector for Azure to the Azure API for FHIR data plane. Also, the Authority on the Authentication page of your Azure API for FHIR should NOT be changed from the Azure Active Directory in your subscription, or this connection will be broken.
```

Comment:
"configure Azure RBAC for your Azure API for FHIR ... using the Managed Identity ID output by the ARM deployment" - Is this actually possible? I have tried to use the Managed Identity ID to set up RBAC, but when I enter the value in the search box, the identity is not listed. I have only been able to search using the name of the App Service Plan.

Reply:
Ahh good catch, it can be done via powershell through managed identity id, but that link only shows the portal setup and you are right, you can only search by name there. I'll update this to reflect the name of the app service. I don't think we need to output that though.